### ISSUE TITLE
Add cross-env to ensure NODE_ENV works on Windows

### ISSUE THREAD

**Issue by mor10** [wprig/wprig#494]
Add [`cross-env`](https://www.npmjs.com/package/cross-env) to allow `NODE_ENV:production` to not cause Windows computers to have a complete meltdown.

Also requires a small update to `package.json`:

```
"bundle": "cross-env NODE_ENV=production gulp bundleTheme",
```

**Comment by ataylorme** [comment]
@mor10 can you make a pull request?

**Comment by mor10** [comment]
fixed in `v2.0`. Closing.
